tensorrt_llm/_torch/autotuner.py

```
@@ -1085,61 +1217,103 @@ def _profile_runners(
+            tac = valid_tactics[0]
+            if "do_preparation" in runner_arg_names:
+                runner(input_tensors, tactic=-1, do_preparation=True, **kwargs)
+            try:
+                with nvtx_range(f"r{runner_id}, tactic {tac} (single-pair)"):
+                    runner(input_tensors, tactic=tac, **kwargs)
+                best_runner_id, best_tactic = runner_id, tac
+                # No comparable measurement exists; min_time stays 0.0 to
+                # distinguish "recorded without profiling" from a real
+                # timing.
+                min_time = 0.0
+            except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)